src/dps-validation-service/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/dfcms/ValidationController.java

```diff
@@ -43,7 +43,7 @@ public ValidationController(CaseService caseService) {
      *  @param driversLicense
      * @param surcode
      */
-    @RequestMapping(value = "/getValidOpenDfcmsCase", produces = MediaType.APPLICATION_XML_VALUE, method = RequestMethod.GET)
+    @RequestMapping(value = "/getValidOpenDFCMSCase", produces = MediaType.APPLICATION_XML_VALUE, method = RequestMethod.GET)
```

Comment:
Hi Alex,
On WebMethods, its 'GetValidOpenDFCMCase' instead of 'getValidOpenDFCMSCase'. G' in 'get' is uppercase as well. Does this matter?
Thanks.

Reply:
@alexjoybc

Reply:
yes it does, thanks @karthikmsqa